### ISSUE TITLE
KeyError: 'GPS GPSLatitude'

### ISSUE THREAD

**Issue by WangQi1024** [xingag/spider_python#3]
Can you tell me how to solve it? Thanks.

**Comment by WangQi1024** [comment]
Indeed,there are no str 'GPS GPSLatitude' in img_exif,only like this:
img_exif
{'EXIF ColorSpace': (0xA001) Short=sRGB @ 48, 'EXIF ExifImageLength': (0xA003) Long=1920 @ 72, 'Image Orientation': (0x0112) Short=Horizontal (normal) @ 18, 'Image ExifOffset': (0x8769) Long=38 @ 30, 'EXIF ExifImageWidth': (0xA002) Long=1080 @ 60}

**Comment by Chiyo1112** [comment]
读一读代码，保存那个.py文件

**Comment by Tina-Li402** [comment]
i have the same problem, did you solve it ? i am confused

**Comment by JustinWjq** [comment]
i have the same problem too.

**Comment by Yijing998520** [comment]
i have the same problem too.
